# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/HyperTextAccessible.cpp
# Commit: 8d874e7a2a8f
# Full Hash: 8d874e7a2a8fe3def139cce04a725ecabe0e21f0
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-06-29 16:48:50
# Regressor Bug: 1757121
# File Overlap Count: 1
# Description:
#   Bug 1757121 part 1: Allow nsAccessibilityService::MarkupAttribute/MarkupAttributes to take an Accessible. r=morgan
#   
#   Previously, these methods took a DOM nsIContent, making it impossible for them to work for RemoteAccessible.
#   GetMarkupMapInfoForNode has been renamed to GetMarkupMapInfoFor and has overloads for nsIContent and Accessible.
#   MarkupAttribute is a template which can take either, as we still need the nsIContent version elsewhere.
# ==============================================================================

diff -r 1d98c5a7861b -r 8d874e7a2a8f accessible/generic/HyperTextAccessible.cpp
--- a/accessible/generic/HyperTextAccessible.cpp	Wed Jun 29 06:55:45 2022 +0000
+++ b/accessible/generic/HyperTextAccessible.cpp	Wed Jun 29 07:04:12 2022 +0000
@@ -1406,7 +1406,7 @@
   }
 
   if (HasOwnContent()) {
-    GetAccService()->MarkupAttributes(mContent, attributes);
+    GetAccService()->MarkupAttributes(this, attributes);
     if (mContent->IsMathMLElement()) SetMathMLXMLRoles(attributes);
   }
 